vapi/tags/categories.go

```diff
@@ -91,7 +91,10 @@ func (c *Manager) CreateCategory(ctx context.Context, category *Category) (strin
 	}
 	url := c.Resource(internal.CategoryPath)
 	var res string
-	return res, c.Do(ctx, url.Request(http.MethodPost, spec), &res)
```

Comment:
+1 for changing the cases where we return a response value (`var res string` here). But don't think we need to change anything when just return an error (see comments below)